src/Chronos.php

```diff
@@ -182,6 +183,66 @@ public static function hasTestNow(): bool
         return static::$testNow !== null;
     }
 
+    /**
+     * Create a new DateInterval instance from specified values.
+     *
+     * @param int|null $years The year to use.
+     * @param int|null $months The month to use.
+     * @param int|null $weeks The week to use.
+     * @param int|null $days The day to use.
+     * @param int|null $hours The hours to use.
+     * @param int|null $minutes The minutes to use.
+     * @param int|null $seconds The seconds to use.
+     * @param int|null $microseconds The microseconds to use.
+     * @return \DateInterval
+     */
+    public static function createInterval(
```

Comment:
This is something we can backport reasonably easy. We could also deprecate the entire `ChronosInterval` as well.

Reply:
Yes, this should be easy to backport.